.github/workflows/test-helpers/get-issue-validation-log

```diff
@@ -39,7 +39,7 @@ function download-and-extract {
   actual=$(grep 'Validating issue #[0-9]*\.[^"]' log | sed -Ee 's/.*#([0-9]*)\..*/\1/')
   echo "Expected: ${expected}"
   echo "Actual:   ${actual}"
-  test $actual = $expected
+  test "$actual" = "$expected"
```

Comment:
nit: This was failing with a less-than-helpful error message if one of the args was empty.